tests/ceph_installer/test_ansible.py

```diff
@@ -146,6 +147,22 @@ def run(ceph_cluster, **kw):
 
     LOG.info("Ceph ansible version " + ceph_installer.get_installed_ceph_versions())
 
+    # Check if ansilbe.log file has any logs initially
+    if custom_log:
+        ansible_log_file, _ = ceph_installer.exec_command(
+            cmd=f"cat {ansible_dir}/ansible.cfg | grep log_path | cut -d \" \" -f3 | tr -d '\n\r'".format(
```

Comment:
This would be
```
cmd=f"grep log_path {ansible_dir}/ansible.cfg | cut -d ' ' | tr -d '\n\r'"
```

Reply:
@psathyan, thanks, I got confused last time! I now corrected the changes! Will test them and post them!